leftwm-core/src/utils/command_pipe.rs

```diff
@@ -441,4 +470,24 @@ mod test {
             Command::MoveWindowToPreviousTag { follow: true }
         );
     }
+
+    #[test]
+    fn build_focus_next_tag_without_parameter() {
+        assert_eq!(
+            build_focus_next_tag(""),
+            Command::FocusNextTag {
+                behavior: command::FocusDeltaBehavior::Default
+            }
+        )
```

Comment:
To make github clippy happy:
```suggestion
        );
```